web-animations-1/Overview.bs

```diff
@@ -2788,23 +2868,70 @@ Animation effects {#animation-effects}
   </div>
 
 
-Fill behavior {#fill-behavior}
--------------
+### Fill behavior and fill modes ### {#fill-behavior}
 
   The effect of an [=animation effect=] when it is not [=in play=]
   is determined by its <dfn>fill mode</dfn>.
+  The effect of each [=fill mode=] is described below:
 
-  The possible [=fill modes=] are:
+  <dl dfn-for="fill mode" id="fill-modes">
+  : <dfn>none</dfn>
+  ::
+      The animation effect has no effect when it is not [=in play=].
 
-  *   none,
-  *   forwards,
-  *   backwards, and
-  *   both.
+  : <dfn>forwards</dfn>
+  ::
+      When the animation effect is in the [=animation effect/after phase=],
+      the animation effect will produce the same [=iteration progress=] value
+      as the last moment it is scheduled to be [=in play=].
+
+      For all other times that the animation effect is not [=in play=],
+      it will have no effect.
+
+  : <dfn>backwards</dfn>
+  ::
+      When the animation effect is in the [=animation effect/before phase=],
+      the animation effect will produce the same [=iteration progress=] value
+      as the earliest moment that it is scheduled to be [=in play=].
+
+      For all other times that the animation effect is not [=in play=],
+      it will have no effect.
+
+  : <dfn>both</dfn>
+  ::
+      When the animation effect is in its [=animation effect/before phase=],
+      [=fill mode/backwards=] fill behavior is used.
+
+      When the animation effect is in its [=animation effect/after phase=],
+      [=fill mode/forwards=] fill behavior is used.
+
+  </dl>
 
-  The normative definition of these modes
+  The normative effect of these modes
   is incorporated in the calculation of the [=active time=]
   in [[#calculating-the-active-time]].
 
+  Some examples of the these [=fill modes=] are illustrated below.
```

Comment:
Should this part onwards be in a non-normative block?

Reply:
It's pretty clearly described as an example, so I don't think it's necessary, unless you really want a green box here.

Reply:
I am pretty partial to a bit of green but you know better than I do what's standard practice in these specs so if you think it's fine without, I'm sure it's fine.